Software/javascript/README.md

```diff
@@ -2,16 +2,46 @@
 <!-- press ctrl+K then V to open a preview of the MarkDown file  -->
 =======
 >>>>>>> Stashed changes
-# Page Main Title 
-short description of the directory's subject
+# Vanilla JavaScript 
+Here you will find Vanilla JavaScript CheatSheats, guides, articles, etc...
 
-# Table Of Contents
```

Comment:
you are missing a table of contents.
it is useful to have so people can find what they are looking for.
use this format: 
````
[header-title](#header-text) 
````

the header title does'nt have to match the header exactly, the header text must. use "-" for spaces.
see a good example [here](https://github.com/ItsDrTsach/together-we-strong-c4s/tree/master/Software/projects)